examples/register-client-request.raml

```diff
@@ -0,0 +1,8 @@
+#%RAML 1.0 register client example
+client_name:	Example App
```

Comment:
Tab characters don't seem to be allowed in YAML (or at least cause parsing errors in the test suite)